### PR TITLE
Feature: Add JS and WASM support to the Compose extensions

### PR DESCRIPTION
**Pull request by jordond**

### CODE REVIEW

File: extensions/colormath-ext-jetpack-compose/build.gradle.kts [hunk 1]
```diff
@@ -18,6 +18,17 @@ kotlin {
 
     jvm()
 
+    js {
+        browser()
```
Comment (ajalt) [inline extensions/colormath-ext-jetpack-compose/build.gradle.kts:22]: We don't run browser tests, so no need for `browser()`

File: extensions/colormath-ext-jetpack-compose/build.gradle.kts [hunk 2]
```diff
@@ -18,6 +18,17 @@ kotlin {
...
+        nodejs()
+    }
+
+    @Suppress("OPT_IN_USAGE")
```
Comment (ajalt) [inline extensions/colormath-ext-jetpack-compose/build.gradle.kts:26]: Let's use `@OptIn(ExperimentalWasmDsl::class)` instead. Let's match how we do the declarations for the [main module][1]

[1]: https://github.com/ajalt/colormath/blob/9ff469060467d478466315280c19d803e4dd2bcd/colormath/build.gradle.kts#L14-L17

**Review by ajalt — CHANGES_REQUESTED** [review]

**Comment by ajalt** [comment]
Thanks! Just a couple of comments.

**Comment by jordond** [comment]
Addressed the comments 👍

**Comment by ajalt** [comment]
Ah, looks like compose-ui didn't publish a wasm artifact [until 1.6.0][1], which is still in beta. So I guess this PR will have to wait until they release 1.6.0.

[1]: https://repo1.maven.org/maven2/org/jetbrains/compose/ui/ui-graphics-wasm-js/

**Comment by jordond** [comment]
Oh shoot, yeah I forgot about that. Hoping for a stable 1.6 soon 🤞

**Pull request closed without merging**